alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -471,8 +472,27 @@ void RunningPlan::deactivate()
     } else {
         _ae->getTeamObserver().notifyRobotLeftPlan(_activeTriple.abstractPlan);
     }
+
     revokeAllConstraints();
     deactivateChildren();
```

Comment:
shouldn't the onTerminate job for this running plan have a prerequisite that first level children onTerminate should be called? You are only getting requisites for siblings.